[MTM-50975] Release notes: optional configuration in SSO page, allowi… (#2328)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-10-20-499-0-single-sign-on-improve-error-information-page.md

---
date: ""
title: Single-sign-on improve error information page
product_area: Platform services
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
component:
  - value: q3kclF6pO
    label: Authentication
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-50975
version: 10.20.499.0
---

> SSO configuration has been extended with the optional: `Redirect to the user interface application`.

Reviewer: Previously, when an error occurred on logging in via SSO, the plain HTML error text was displayed in the browser. With this change, optional configuration has been added which allows displaying the error text as a standard UI error message. The new configuration option is optional and does not affect current SSO configurations.

Reviewer: I would change it to the following text for the users and have the details as a change log in the operations release notes. What do you think, @pink-mongoose @mbak-c8y?

Author: I do not think Operations release is required, it is rather tenant administrator change.

Author: I rephrased it based on comment, @BeateRixen  is it sound better now?

---
date: ""

> title: Single-sign-on improve error information page

Reviewer: ```suggestion
title: Improved error information for single sign-on login
```

product_area: Platform services
change_type:
  - value: change-2c7RdTdXo4

>     label: Improvement

Reviewer 2: Isn't this a feature and not improvement?

Author: improvement

component:
  - value: q3kclF6pO
    label: Authentication
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-50975
version: 10.20.499.0
---
Previously, when an error occurred on logging in via SSO, the plain HTML error text was displayed in the browser.
With this change, optional `Redirect to the user interface application` configuration has been added which allows displaying the error text as a standard UI error message.

> The new configuration option is optional and does not affect current SSO configurations.

Reviewer: ```suggestion
The new configuration parameter is optional and does not affect current SSO configurations.
```

> Using this option requires updating the "Valid Redirect URIs" in the authorization server with the value "<tenant_domain>/apps/*".

Reviewer: ```suggestion
Using this option requires updating the "Valid Redirect URIs" in the authorization server with the value "<tenant_domain>/apps/*". For details, refer to [Custom template configuration](/authentication/sso/custom-template).
```
